README.md

```diff
@@ -689,10 +689,15 @@ with .NET tasks inside of workflows:
 * Do not use `Task.Delay`, `Task.Wait`, timeout-based `CancellationTokenSource`, or anything that uses .NET built-in
   timers.
   * `Workflow.DelayAsync`, `Workflow.WaitConditionAsync`, or non-timeout-based cancellation token source is suggested.
+* Do not use `Task.WhenAny`.
+  * Use `Workflow.WhenAnyAsync` instead.
```

Comment:
Does this method require the `Async` suffix? Wouldn't it be better to match the `Task` API naming?

Reply:
I chose `Workflow.DelayAsync` even though `Task.Delay`, so I did the same here. I think following [VSTHRD200](https://github.com/microsoft/vs-threading/blob/main/doc/analyzers/VSTHRD200.md) everywhere is better than trying to match `Task` name exactly for the couple of alternatives we're providing.

Reply:
I don't think we should (and didn't when we introduced `DelayAsync`). In fact, I suspect .NET regrets not suffixing these with `Async` back when they wrote it.